src/classes/content/wordpress/class-abstract-wordpress-content-service.php

```diff
@@ -102,11 +114,10 @@ public function set_about_jsonld( $content_id, $value ) {
 		if ( null === $value ) {
 			return $wpdb->query(
 				$wpdb->prepare(
-					"
-			UPDATE {$wpdb->prefix}wl_entities
-			SET about_jsonld = NULL
-			WHERE content_id = %d AND content_type = %d
-			",
+					"UPDATE {$wpdb->prefix}wl_entities
+					SET about_jsonld = NULL, match_name = %s
+					WHERE content_id = %d AND content_type = %d",
+					$match_name,
```

Comment:
I think this is literally setting a `"NULL"` string (not the `NULL` value):
https://github.com/insideout10/wordlift-plugin/pull/1707/files#diff-06d27e9e3967581feb2e9bea7050a7cfdcab05d82a8e7e6de5543f5e2acecc65R82

Reply:
yes, correct, due to this WP settings https://github.com/insideout10/wordlift-plugin/blob/43be605e50bbf7bb54b54743a065bfe5c6585824/src/classes/content/wordpress/class-abstract-wordpress-content-service.php#L109 need to pass string "NULL"

Reply:
yes, correct, due to this WP settings https://github.com/insideout10/wordlift-plugin/blob/43be605e50bbf7bb54b54743a065bfe5c6585824/src/classes/content/wordpress/class-abstract-wordpress-content-service.php#L109 need to pass string "NULL"